### ISSUE TITLE
cmake_minimum_required is too low for features used

### ISSUE THREAD

**Issue by basvandertol** [lofar-astron/DP3#309]
Commits 2401a19c and 194c2389 (development branch) introduce the usage of the cmake command `file(CREATE_LINK ...)` 
This feature was only introduced in CMake 3.14, while the cmake_minimum_required version set in the toplevel CMake file is  3.9

Either do not use this feature, or update the minimum required version accordingly.

```
./DDECal/test/integration/CMakeLists.txt:file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}/source
./DPPP/test/integration/CMakeLists.txt:file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR}/source

```

**Comment by mhardcastle** [comment]
Agreed, please remove use of this feature, it breaks build compatibility with stock Debian buster base images.

**Comment by aroffringa** [comment]
This is to be fixed in https://git.astron.nl/RD/DP3/-/merge_requests/295 .

@mhardcastle unless you need recent features, which I suspect is not the case, you don't have to use master - we generally recommend to use only released versions for 'production' like usage.

**Comment by mhardcastle** [comment]
Yes, I have backed off to v4.2.0 now. Using master does allow me to find out what's coming down the track that might cause problems later!

**Comment by aroffringa** [comment]
True -- and actually I highly appreciate bug reports before new versions are released :).
